src/general/MainMenu.cs

```diff
@@ -752,6 +788,47 @@ private void WarnAboutNoEnabledMods()
         }
     }
 
+    private float TrackMenuPerformance()
```

Comment:
I think tracking menu performance is a good term to describe what this does. I already suggested swapping to the Godot engine inbuilt way of getting the current framerate, so that would make this method less technical towards framerate calculation...

Reply:
It is used to update averageFramerate though, and then averageFramerate is used as the performance metric. It's mostly unimportant anyway, just to stop anyone else rerolling an average FR function when it's already here.

Reply:
Well it's not really useful for general use at all if it is just in the main menu class...

Reply:
Haha yeah that's true I guess. :)